app/controllers/api/v1/moves_actions.rb

```diff
@@ -139,5 +141,19 @@ def active_record_relationships
     def supported_relationships
       MoveSerializer::SUPPORTED_RELATIONSHIPS
     end
+
+    def create_auto_event(move, event_class, details = {})
```

Comment:
Would be nice to move this method to the moves controller, as it appears to be exactly the same in both of these modules so would be better to just define it once.

Reply:
@martyn-w After further thought, I'm tempted to move this to the api controller as there is very similar code for the person escort record controller too. I'm happy to do that work if you want to leave this as is for now.